backend/src/process.py

```diff
@@ -177,9 +179,28 @@ async def __process(self, node: UsableData) -> Any:
             # Run the node and pass in inputs as args
             run_func = functools.partial(node_instance.run, *enforced_inputs)
             output = await self.loop.run_in_executor(None, run_func)
+            node_outputs = node_instance.get_outputs()
+            broadcast_data = dict()
+            if len(node_outputs) > 0:
+                output_idxable = [output] if len(node_outputs) == 1 else output
+                for idx, node_output in enumerate(node_outputs):
+                    try:
+                        broadcast_data[node_output.id] = node_output.get_broadcast_data(
+                            output_idxable[node_output.id or 0]
+                        )
```

Comment:
1. Output values are accessed by index.
2. The output id defaults to the index if `None`.

```suggestion
                        output_id = node_output.id if node_output.id is not None else idx
                        broadcast_data[node_output.id] = node_output.get_broadcast_data(output_idxable[idx])
```